dipy/utils/tests/test_omp.py

```diff
@@ -27,6 +30,7 @@ def test_set_omp_threads():
         assert_equal(cpu_count(), 1)
 
 
+@pytest.mark.skipif(has_joblib or has_ray)
```

Comment:
Arf, I reviewed too fast.  There is a issue here and CI's are failing because of that. it is missing a reason:

```diff
- Error evaluating 'skipif': you need to specify reason=STRING when using booleans as conditions.
```

Reply:
Fixed in e0986ecd3